package/entrypoint.sh

```diff
@@ -6,6 +6,18 @@ if [ ! -e /run/secrets/kubernetes.io/serviceaccount ] && [ ! -e /dev/kmsg ]; the
     exit 1
 fi
 rm -f /var/lib/rancher/k3s/server/cred/node-passwd
+if [ -e /var/lib/rancher/etcd ] && [ ! -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  mkdir -p /var/lib/rancher/k3s/server/db
+  ln -sf /var/lib/rancher/etcd /var/lib/rancher/k3s/server/db/etcd
+  echo -n 'default' > /var/lib/rancher/k3s/server/db/etcd/name
+fi
+if [ -e /var/lib/rancher/k3s/server/db/etcd ]; then
+  k3s server --cluster-init --cluster-reset &> ./k3s-cluster-reset.log
+  if [$? -ne 0]; then
```

Comment:
I'm pretty sure you'll need whitespace around this
```suggestion
  if [ $? -ne 0 ]; then
```